manifest.webapp

```diff
@@ -72,13 +72,20 @@
     "categorization": {
       "type": "node",
       "file": "categorization.js",
+      "comment": "This service is called manually, this is why the trigger is in the past",
       "trigger": "@at 2000-01-01T00:00:00.000Z"
     },
     "stats": {
       "type": "node",
       "file": "stats.js",
       "trigger": "@event io.cozy.bank.operations:CREATED,UPDATED",
       "debounce": "1m"
+    },
+    "autogroups": {
+      "type": "node",
+      "file": "groups.js",
+      "trigger": "@event io.cozy.bank.accounts:CREATED",
+      "debounce": "1m"
```

Comment:
Ping @fffflo . This means you would put the call inside the banking konnectors. I am not really comfortable with this since a konnector can easily forgot this responsibility. 

An alternative would be to put a shorter debounce time like 10s. All accounts are created within 10s anyway inside banking connectors.

Reply:
Yes a shorter debounce may be the solution. 

We can also use no debounce at all. That's what services like cozy-home's `myselfFromIdentities` do: https://github.com/cozy/cozy-home/blob/master/manifest.webapp#L132-L136. But if a konnector creates multiple accounts in a very short time interval, we may have a performance problem by running n times the service very quickly?

Reply:
What does no debounce mean anyway ? Because there is still a queue on redis side that is not synchronous. So you would still potentially have a delay that can be of several seconds. That removes the advantage of using no debounce to me.

Reply:
I didn't thought about that but you're right. The best solution here would be a small debounce I think.

Reply:
I put a debounce of 5s.